app/portainer/components/form-components/FormControl/FormControl.tsx

```diff
@@ -25,13 +25,13 @@ export function FormControl({
       <div className={clsx('form-group', styles.container)}>
         <label
           htmlFor={inputId}
-          className="col-sm-3 col-lg-2 control-label text-left"
+          className="col-sm-4 col-lg-3 control-label text-left"
```

Comment:
this will change everyone who uses FormControl. I wish this would just automatically calculate the needed width for the label. what about defaulting to `col-sm-3 col-lg-2` but receiving an optional `labelWidthClass`, and `inputWidthClass`?

or maybe receive size for label? `small | medium | large`?